docs/src/code-samples/examples/input-date-fns.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+
+import format from 'date-fns/format';
+import parse from 'date-fns/parse';
+
+export default function Example() {
+  const FORMAT = 'M/D/YYYY';
+  return (
+    <DayPickerInput
+      formatDate={format}
```

Comment:
However, date-fns `format` and `parse`  does not follow the `parseDate` and `formatDate` function signature.

This should be the right way to use the `formatDate` attribute

```jsx
formatDate = { (day, format, locale) => format(day, format, { locale }) }
```

Too bad `formatDate` uses the locale as third argument! 🤦‍♂️  Well, we could update them in a next release...